Out of bounds array bug

Hi Armando,

I am seeing a compilation warning for the following two lines:

https://github.com/vasole/fisx/blob/c8373b80132132012a81d52eab0281169da157b3/src/fisx_math.cpp#L152-L153

```  
/tmp/pip-req-build-8m4M4E/src/fisx_math.cpp: In static member function ‘static double fisx::Math::AS_5_1_56(const double&)’:
/tmp/pip-req-build-8m4M4E/src/fisx_math.cpp:152:25: warning: array subscript is above array bounds [-Warray-bounds]
             num = (num + a[i]) * x;
                             ^
/tmp/pip-req-build-8m4M4E/src/fisx_math.cpp:153:25: warning: array subscript is above array bounds [-Warray-bounds]
             den = (den + b[i]) * x;
```

This looks like a bug to me as the first iteration will start at 4 causing therefore the out of bounds array access.

If you agree, I will gladly submit a PR with a fix.

Best,

Tom

--- a/src/fisx_math.cpp
+++ b/src/fisx_math.cpp
@@ -147,7 +147,7 @@ double Math::AS_5_1_56(const double & x)
     }
     num = x;
     den = x;
-    for (int i = 4; i > 0; --i)
+    for (int i = 3; i >= 0; --i)
     {
         num = (num + a[i]) * x;
         den = (den + b[i]) * x;

